### PR TITLE
[StickyHeader] to Nuget.

### PR DESCRIPTION
**Pull request by adlair**

### CODE REVIEW

File: Android/StickyHeader/component/component.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 id: stickyheader
 name: Sticky Header
-version: 1.1.1
+version: 1.11
```
Comment (bholmes) [inline Android/StickyHeader/component/component.yaml]: Why did this change to 1.11?  Is this a typo?

Comment (bholmes) [reply Android/StickyHeader/component/component.yaml]: I am thinking that 1.1.2 may be better.

File: Android/StickyHeader/nuget/Xamarin.StickyHeader.nuspec [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  <metadata>
+    <id>Xamarin.StickyHeader</id>
+    <title>StickyHeader for Xamarin.Android</title>
+    <version>1.11</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>A very simple library that allows you to stick any View as a header of a ListView, RecyclerView or ScrollView.</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <projectUrl>https://components.xamarin.com/view/stickyheader</projectUrl>
```
Comment (bholmes) [inline Android/StickyHeader/nuget/Xamarin.StickyHeader.nuspec]: Use fwlinks and point back to github.  Be sure and move license file from components folder to root of the project.  `../`

Comment (newky2k) [reply Android/StickyHeader/nuget/Xamarin.StickyHeader.nuspec]: This is the link to the doc detailing it -> https://gist.github.com/bholmes/39bad5c2296fe445df9e5bbf9261ab15

File: Android/StickyHeader/nuget/Xamarin.StickyHeader.nuspec [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    <licenseUrl>https://components.xamarin.com/license/stickyheader</licenseUrl>
```
Comment (newky2k) [inline Android/StickyHeader/nuget/Xamarin.StickyHeader.nuspec]: Use fwlinks as specified here -> https://gist.github.com/bholmes/39bad5c2296fe445df9e5bbf9261ab15

File: Android/StickyHeader/nuget/Xamarin.StickyHeader.nuspec [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  </metadata>
+  <files>
+    <file src="output/unified/StickyHeader.dll" target="lib/Xamarin.iOS" />
```
Comment (newky2k) [inline Android/StickyHeader/nuget/Xamarin.StickyHeader.nuspec]: Need to as a external-deps file

**Review by newky2k — CHANGES_REQUESTED** [review]
Component.yaml file need to include is_shell: true, samples need to have nobuild: true.  it also needs the external deps file

File: Android/StickyHeader/component/component.yaml [hunk 5]
```diff
@@ -17,6 +17,10 @@ icons:
 samples: 
   - name: Android Sample
     path: ../samples/StickyHeaderSample.sln
+    installNuGets:
```
Comment (newky2k) [inline Android/StickyHeader/component/component.yaml]: missing the remove part of the installNugets tag

File: Android/StickyHeader/component/component.yaml [hunk 6]
```diff
@@ -17,6 +17,10 @@ icons:
...
+      - project: StickyHeaderSample
+        packages:
+          - Xamarin.StickyHeader
```
Comment (newky2k) [inline Android/StickyHeader/component/component.yaml:25]: missing the local-nuget-repo tag

Comment (newky2k) [reply Android/StickyHeader/component/component.yaml:25]: missing the is_Shell tag

Comment (newky2k) [reply Android/StickyHeader/component/component.yaml:25]: Missing the external-dependencies file in the component

**Review by newky2k — CHANGES_REQUESTED** [review]

**Comment by newky2k** [comment]
Fails to compile with the following error -> https://gist.github.com/newky2k/ab10bd46322babd8aa0ad9cae6c77ce8

File: Android/StickyHeader/nuget/Xamarin.StickyHeader.nuspec [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>A very simple library that allows you to stick any View as a header of a ListView, RecyclerView or ScrollView.</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <projectUrl>https://go.microsoft.com/fwlink/?linkid=869732</projectUrl>
+    <licenseUrl>https://go.microsoft.com/fwlink/?linkid=869731</licenseUrl>
+  </metadata>
+  <files>
+    <file src="output/unified/StickyHeader.dll" target="lib/Xamarin.iOS" />
```
Comment (newky2k) [inline Android/StickyHeader/nuget/Xamarin.StickyHeader.nuspec]: The output folder of the dll, isnt  into the unified folder within output, its just into output

**Review by newky2k — CHANGES_REQUESTED** [review]

**Comment by adlair** [comment]
I added changes in the next PR: https://github.com/xamarin/XamarinComponents/pull/361.

**Comment by bholmes** [comment]
There was a problem with Git and GitHub where the PR was disconnected from the fork.  This PR is patched and opened again in #361.

**Pull request closed without merging**